docs/user_guide/mitigation.rst

```diff
@@ -527,16 +555,25 @@ Group :code:`"a"` has an average loss of :math:`0.05`, while group
 :code:`"b"`'s average loss is :math:`0.5`.
 
 .. doctest:: mitigation
+    :options:  +NORMALIZE_WHITESPACE
 
     >>> from fairlearn.reductions import BoundedGroupLoss, ZeroOneLoss
-    >>> from fairlearn.metrics import mean_absolute_error_group_summary
+    >>> from sklearn.metrics import mean_absolute_error
     >>> bgl = BoundedGroupLoss(ZeroOneLoss(), upper_bound=0.1)
     >>> X                  = np.array([[0], [1], [2], [3]])
     >>> y_true             = np.array([0.3, 0.5, 0.1, 1.0])
     >>> y_pred             = np.array([0.3, 0.6, 0.6, 0.5])
     >>> sensitive_features = np.array(["a", "a", "b", "b"])
-    >>> mean_absolute_error_group_summary(y_true, y_pred, sensitive_features=sensitive_features)
-    {'overall': 0.275, 'by_group': {'a': 0.0499..., 'b': 0.5}}
+    >>> mae_summary = MetricFrame(mean_absolute_error,
```

Comment:
Nit: do we still want to call it summary even though the name changed?

Reply:
Adjusted.